src/js/components/Data/stories/Controlled.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
```

Comment:
Clicking 'reset filters' doesn't do anything after the filters have already been applied. If this is the intended behavior it feels like we don't need to display the button in this case.

https://user-images.githubusercontent.com/54560994/207737878-9be670f4-e1b2-49ad-8dba-4081e70afcd3.mp4

Reply:
I also wondered the same if that was intentional or it should clear the options if clicked again

Reply:
The tricky part of this is that if we don't show the button, then dynamically adding it causes some unfortunate re-layout behavior, either the Drop needs to change size or the buttons end up looking like:
<img width="249" alt="Screenshot 2022-12-19 at 1 04 58 PM" src="https://user-images.githubusercontent.com/11637956/208523767-5fd49498-5f6f-4740-9474-c107213f9a02.png">
And, we generally don't want disabled buttons.
Perhaps we leave it there to take the space up but don't have it be visible and don't let keyboard traversal or screen readers see it but allow it to reserve the space it wants?

Reply:
Changed to disable and hide the button to preserve the layout.